students/shibin_mathew/lesson9/mailroom.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python3
+import sys
+from collections import defaultdict
+
+
+class Donor:
+    def __init__(self, names, amount):
+        self.names = names
+        self.amount = amount
+
+    @property
+    def names(self):
+        return "{}".format(self._names)
```

Comment:
don't need this as a property, `_names` is already a string so no need to format.

also, it shouldn't be `names`, but a single `name`